explainer.md

```diff
@@ -301,27 +296,17 @@ Badge.clear();
 ```
 
 ## UX treatment
-Badges should appear as a user agent defined overlay on top of the app's icon, about a quarter of the size, and in one of the four corners. If the exact representation of a badge is not supported (e.g., a 2-digit number while only single characters are allowed, or a character where only numbers are allowed) the user agent should make a best effort attempt to map the unsupported data onto supported data. This may involve:
+Badges may appear in any place that the user agent deems appropriate. In general, these places should be obviously related to the pages being badged, so users understand what the status is for. Appropriate places could include:
+- Tab favicons.
+- Bookmark icons.
+- [OS Specific Contexts](#OS-Specific-Contexts) for [Installed Web Applications](https://www.w3.org/TR/appmanifest/#installable-web-applications).
+  
+> Note: When showing an badge in an [OS Specific Context](#OS-Specific-Contexts) user agents should attempt reuse existing [operating system APIs and conventions](#Specific-operating-system-treatment-for-installed-web-applications), to achieve a native look-and-feel.
+
+If the exact representation of a badge is not supported (e.g., a 2-digit number while only single characters are allowed, or a character where only numbers are allowed) the user agent should make a best effort attempt to map the unsupported data onto supported data. This may involve:
 - Saturating a number: 351 -> '99+'
 - Degrading the data: 7 -> 'flag' when only flag badges are supported.
 
-### Badging Documents
-User agents may choose to apply the badge in other places they display a favicon for the site. These might include:
-- The favicon displayed on tabs
-- Bookmark Icons
-
-### Badging Installed Web Applications
```

Comment:
Did this section get moved anywhere? It seems important.

Reply:
First bullet is covered by [OS Specific Contexts](https://github.com/WICG/badging/blob/794c8b9331c1e12fe26036c1025860e609af3e94/explainer.md).

Second bullet is covered by the new note (see line 306 above).

Degrading the data is already covered above (and again, implicitly by saying to follow OS conventions, and listing the Android conventions are flag only).